canonical-websites/design.ubuntu.com
# Add social icons to footer

## Done

* added svg icons until Vanilla pattern is released
* updated —watch to use auto-prefixer

## QA

- download branch
- `./run serve`
- look at the footer on any [page](http://0.0.0.0:8011/brand/)
- icons should link to ubuntu designers twitter, instagram and blog

## Issue / Card

Fixes #17

## Screenshots

![image](https://user-images.githubusercontent.com/441217/30690453-35dd9acc-9ebc-11e7-9595-97a90e8401b7.png)
